conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -428,6 +446,10 @@ private void detachBitmovinEventListeners() {
     private synchronized void transitionState(ConvivaSdkConstants.PlayerState state) {
         Log.d(TAG, "Transitioning to :" + state.name());
         convivaVideoAnalytics.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.PLAYER_STATE, state);
+        if (bitmovinPlayer.isAd()) {
+            Log.d(TAG, "Transitioning ad state to: " + state.name());
+            convivaAdAnalytics.reportAdMetric(ConvivaSdkConstants.PLAYBACK.PLAYER_STATE, state);
+        }
```

Comment:
Is it right so change the state for the ad and the main content analytics? or should this be a either-or?

Reply:
As main content tracking is `Untracked` between ad break started and ad break finished this should be an none-issue. Checking Touchstone also shows that this is fine.